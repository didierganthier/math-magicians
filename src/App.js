import './App.css';
import Calculator from './components/Calculator';

function App() {
  return (
    <div>
      <Calculator />
    </div>
  );
}

export default App;
