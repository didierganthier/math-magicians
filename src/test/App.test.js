import { render, screen } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import App from '../App';

it('testing title', () => {
  render(
    <Router>
      <App />
    </Router>,
  );
  expect(screen.getByText('Math Magicians')).toBeInTheDocument();
});
