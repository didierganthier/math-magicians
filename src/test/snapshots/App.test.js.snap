// Jest Snapshot v1, https://goo.gl/fbAQLP

// Expect hidden h1 element in Quote component
exports[`testing title 1`] = `
    <h1>404</h1>
`;
  